src/utils.js

```diff
@@ -22,6 +24,42 @@ const isInvalidValidPassword = password => {
   }
 };
 
+/**
+ * @desc get hash from dApp
+ * @param {object} content
+ * @return {string} hash string
+ */
+const hashDappContent = content => {
+  let buff = Buffer.allocUnsafe(0);
```

Comment:
can you not use `new Buffer()`?

Reply:
in node 10 new Buffer() is Deprecated , I change it to Buffer.alloc(0); for better